packages/host/app/components/matrix/room-message.gts

```diff
@@ -367,7 +365,7 @@ export default class RoomMessage extends Component<Signature> {
     if (this.operatorModeStateService.patchCard.isRunning) {
       return;
     }
-    let { payload, eventId } = this.args.message.command;
+    let { payload, eventId } = this.args.message.command!;
```

Comment:
we repeat `this.args.message.command!` a lot. in general i'm suspicious of `!` usage. if we _really_ think we are always gonna have a command, then let make a getter that will return a command and throw with a useful exception otherwise so if we don't have a command it will be clear that was the cause of the problem instead of having to hunt thru code to find out what is going on.